index.d.ts

```diff
@@ -112,6 +112,7 @@ type Palette = {
 		sectionTitle: Colour;
 		avatar: Colour;
 		card: Colour;
+		cardInvertLogo: boolean;
```

Comment:
I don't know that this is the right approach, so looking for ideas on how else to handle. The other option is to have some sort of method of determining if the specific background colour is dark enough to  warrant inverting the logo, and put that in the place it's used.